tests/components/bang_olufsen/test_media_player.py

```diff
@@ -689,6 +699,43 @@ async def test_async_select_source(
     assert mock_mozart_client.post_remote_trigger.call_count == video_source_call
 
 
+async def test_async_select_sound_mode(
+    hass: HomeAssistant,
+    mock_mozart_client: AsyncMock,
+    mock_config_entry: MockConfigEntry,
+) -> None:
+    """Test async_select_sound_mode."""
+
+    mock_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+
+    states = hass.states.get(TEST_MEDIA_PLAYER_ENTITY_ID)
+    assert states.attributes[ATTR_SOUND_MODE] == TEST_ACTIVE_SOUND_MODE_NAME
+
+    active_listening_mode_callback = (
+        mock_mozart_client.get_active_listening_mode_notifications.call_args[0][0]
+    )
+
+    await hass.services.async_call(
+        "media_player",
+        "select_sound_mode",
```

Comment:
Please use constants

Reply:
Got it. I'll open a PR for the rest of the tests.